### PR TITLE
[ASM] - RPR Resolution Notes

### PR DESCRIPTION
**Pull request by johnnywilkes**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-hq.paloaltonetworks.local/browse/EXPANDR-2385

## Description
Appending RPR rule name to resolution notes in the case that there is a match

## Screenshots
image

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [X] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

**Comment by johnnywilkes** [comment]
@capanw , please review when possible

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml:3018]: 👍

File: Packs/CortexAttackSurfaceManagement/pack_metadata.json
Comment (capanw) [inline Packs/CortexAttackSurfaceManagement/pack_metadata.json:5]: I am guessing, there is already another PR with 1.5.3?

Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/pack_metadata.json:5]: yes, https://github.com/demisto/content/pull/24837 is using 1.5.3 already

**Review by daryakoval — APPROVED** [review]

**Comment by daryakoval** [comment]
Waiting for ASM approval to merge.

**Review by capanw — APPROVED** [review]

**Pull request merged**
